maestro-client/src/main/resources/maestro-driver-ios-config.xctestrun

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Thats right. I think this is because the script output still points to client. I have fixed it to xcuitest driver. Will be raising PR against this branch.

Reply:
@dmitry-zaitsev you can merge this PR: https://github.com/mobile-dev-inc/maestro/pull/577